package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "jquery-linkselect",
+  "version": "1.0.0",
+  "description": "jquery-linkselect replaces each selected element with a URL-opening form created from all links found within the element.",
+  "main": "jquery.linkselect.js",
+  "repository": "https://github.com/archaichorizon/jquery-linkselect.git",
+  "author": "Steve Hobbs <steve@powder-blue.com>",
```

Comment:
Please can you remove this line, or change it to `Dan Bettles <dan@archaichorizon.com>`, because the package is Archaic Horizon's.  Thanks